.github/workflows/run-tests.yml

```diff
@@ -1,10 +1,8 @@
-name: Run Tests
+name: Run tests
 
 on:
   pull_request:
-  push:
-    branches:
-      - main
```

Comment:
If we're getting rid of this, I think we need to add in the `needs: tests` on the `publish` job in `publish-package.yml`?

Reply:
added